addons/account/views/account_move_views.xml

```diff
@@ -1641,13 +1643,16 @@
                     <filter name="date" invisible="context.get('default_move_type') in ('out_invoice', 'out_refund', 'out_receipt')" string="Accounting Date" date="date"/>
                     <filter name="due_date" string="Due Date" date="invoice_date_due"/>
                     <separator/>
+                    <filter invisible="1" string="My Activities" name="activities_my"
```

Comment:
:socks:  Just to ease grep and understanding stuff (to propagate to other addons)
```suggestion
                    <filter invisible="1" string="My Activities" name="filter_activities_my"
```